subsys/bluetooth/host/hci_core.c

```diff
@@ -140,7 +140,7 @@ static bool drv_quirk_no_reset(void)
 	return  ((bt_dev.drv->quirks & BT_QUIRK_NO_RESET) != 0);
 }
 
-__maybe_unused static bool drv_quirk_no_auto_dle(void)
+__maybe_unused bool bt_drv_quirk_no_auto_dle(void)
```

Comment:
I suspect the __maybe_unused annotation can be removed since it's no longer static (i.e. there shouldn't be any compiler warnings if e.g. Kconfig options result in no references to it)